.github/workflows/continuous-integration.yml

```
@@ -63,7 +63,10 @@ jobs:
 
       - name: run remark on changed files
         run: |
-          readarray -t REMARK_TARGET_FILES <<< $( echo "${{ env.INTERESTING_FILES }}" | grep -e .md$ || true )
+          readarray -t REMARK_TARGET_FILES <<< $(
+            # ppy/osu-wiki#8867 -- disable style checks for regular Markdown files, such as README.md
+            echo "${{ env.INTERESTING_FILES }}" | grep -e .md$ | grep -E -v '^[[:upper:]]+\.md$' || true
```

what kind of wack-ass syntax is this? wouldn't something like this work?

```suggestion
            echo "${{ env.INTERESTING_FILES }}" | grep -e .md$ | grep -E -v '^[A-Z-]+\.md$' || true
```

---

https://www.gnu.org/software/grep/manual/html_node/Character-Classes-and-Bracket-Expressions.html not PCRE, but it works (: can change for readability if you insist

```
/home/user % grep --version
grep (GNU grep) 3.7
--- snip ---
/home/user % echo -en 'a\nb\nDONALD-DUCK\nen.md\nCONTRIBUTING.md' | grep -Ev '^[[:upper:]]+\.md$'
a
b
DONALD-DUCK
en.md
```

---

as is it wouldn't exclude [`TEMPALTE-SERIES.md`](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE-SERIES.md) i'm pretty sure, so may want to add the dash